test/jdk/jdk/net/ExtendedSocketOption/DontFragmentTest.java

```diff
@@ -38,7 +38,10 @@
 
 public class DontFragmentTest {
 
+    private static boolean isMacos;
+
     public static void main(String[] args) throws IOException {
+        isMacos = System.getProperty("os.name").equals("Mac OS X");
```

Comment:
I believe there's a test library class that does that. I never remember what the os.name is supposed to look like.

Reply:
Okay. It's `Platform.isOSX()`